zenoh-plugin-ros2dds/src/dds_types.rs

```diff
@@ -225,6 +227,6 @@ impl From<&DDSRawSample> for ZBuf {
 
 impl From<&DDSRawSample> for Value {
     fn from(buf: &DDSRawSample) -> Self {
-        ZBuf::from(buf).into()
+        Value::new(ZBytes::from(buf), Encoding::default())
```

Comment:
How about using Value::from(value)? You don't need to assign encoding explicitly.

Reply:
So using `Value::from(buf)` here cant return without recursing, 
`fn from(buf: &DDSRawSample) -> Self` at-least according to the rust compiler

Reply:
Oh, I missed that :laughing: